flask_cors/__init__.py

```diff
@@ -342,7 +377,9 @@ def _get_cors_headers(options, request_headers, request_method,
     origin_to_set = _get_cors_origin(options, request_headers.get('Origin'))
 
     if origin_to_set is None:  # CORS is not enabled for this route
+        logger.debug("Could not obtain value of origin header for response, skipping")

```

Comment:
I think this is overly verbose. The log on https://github.com/CoryDolphin/flask-cors/pull/99/files#diff-9510ebf5b86a1db79a60dbe1e9fe2d71R370 should be perfect for this